tests/integration-test/integration_test.go

```diff
@@ -112,7 +112,7 @@ func TestCoordinatorProverInteraction(t *testing.T) {
 	t.Log(version.Version)
 
 	base.Timestamp = time.Now().Nanosecond()
-	coordinatorApp := capp.NewCoordinatorApp(base, "../../coordinator/conf/config.json")
+	coordinatorApp := capp.NewCoordinatorApp(base, "../../coordinator/conf/config.json", "./genesis.json")
```

Comment:
```suggestion
	coordinatorApp := capp.NewCoordinatorApp(base, "../../coordinator/conf/config.json", "../../common/docker/l2geth/genesis.json")
```

Reply:
Can't use the same one, because need set `homesteadBlock` and `istanbulBlock` hard fork number

Reply:
Got it. If wanting to keep only one copy, some settings won't break the l2geth's launch, thus are compatible with other unit tests, e.g.,
```
    "archimedesBlock": 1,
    "shanghaiBlock": 2,
```
(btw, seems that archimedesBlock is not used).

Reply:
<img width="419" alt="image" src="https://github.com/scroll-tech/scroll/assets/6947653/75a60e37-8c83-428b-a5d3-631b36e87b07">

Because the integrate test's block number is 1
hard fork number : [ 1, 2]

prover hardForkNUmber : 1

block --> block number 1
chunk ---> chunk index 1 | start_block_number 1 | end_block_number 1
batch --- > batch index 1 | start_chunk_index 1 | end_chunk_index 1

after : fromBlockNum, toBlockNum := forks.BlockRange(hardForkNumber, bp.forkHeights)
fromBlockNum := 1
endBlockNum := 2

Then bp.chunkOrm.GetChunkByStartBlockNumber(ctx, toBlockNum) to get the the chunk index 1

actually the `istanbulBlock` is not used. But it's not matter.

If don't set `istanbulBlock`, the from fromBlockNum=1 and endBlockNum will be math.MaxInt64